recipes-kernel/linux/linux-poplar_git.bb

```diff
@@ -0,0 +1,82 @@
+require linux.inc
+
+DESCRIPTION = "96boards-poplar kernel"
+
+DEPENDS_append = " dosfstools-native mtools-native u-boot-poplar"
+
+PV = "4.9+git${SRCPV}"
+SRCREV_kernel = "035bdfc09e5c57d85e5b72ac04a588426e44f728"
+SRCREV_FORMAT = "kernel"
```

Comment:
it doesn't seem really useful to use 'kernel' namespace here, since you have just 1 download. it looks like it's a copy/paste from hikey, is there any reason for that?